content-src/components/DiscoveryStreamImpressionStats/ImpressionStats.jsx

```diff
@@ -60,6 +60,35 @@ export class ImpressionStats extends React.PureComponent {
     }
   }
 
+  // This checks if the given cards are the same as those in the last loaded content ping.
+  // If so, it should not send the same loaded content ping again.
+  _needsLoadedContent(cards) {
```

Comment:
Interesting, what's the idea behind this, what is it solving

Reply:
Like, are there multiple events being fired in one tab, or is this for opening multiple tabs and only tracking the first one?